library/common/internal/text_input_model.cc

```diff
@@ -30,13 +30,21 @@ static constexpr char kSelectionIsDirectionalKey[] = "selectionIsDirectional";
 
 static constexpr char kTextKey[] = "text";
 
+static constexpr char kTextInputAction[] = "inputAction";
```

Comment:
Let's add a comment to this block that indicates that these are client configuration keys.